### ISSUE TITLE
Linux - ui hangs after stopping and starting emulation

### ISSUE THREAD

**Issue by dacksm** [citra-emu/citra#4534]
From nightly build 1144 onwards, loading a game from the menu after previously loading a game and stopping it causes the window to hang indefinitely, requiring it to be forcibly killed. The first game to be selected after running citra-qt runs, but stopping emulation, returning to the menu, and picking another game (or the same one again) does not.

This doesn't seem to occur on nightly builds 1142 and earlier ([1143](https://github.com/citra-emu/citra-nightly/releases/tag/nightly-1143) doesn't appear to have a linux build, but I just tried building it from source and it didn't occur there either.) I also have not experienced this issue while running the latest Windows versions.

Specs:
Ubuntu 18.04
Intel Core i5 6200U
Nvidia GeForce 940MX

**Comment by chris062689** [comment]
Are you using flatpak or are you using the binary directly from Github Releases?

**Comment by dacksm** [comment]
It looks like I downloaded citra from  the website before flatpak was put on it, and have the binary installed in a .citra folder

I've been running the earlier builds directly from the Github releases.

**Comment by wwylele** [comment]
It is still strange that the binary doesn't work but the flatpak one works, as there is little difference in code for the two version...

**Comment by dacksm** [comment]
One difference is that the binary follows the system qt theme, while the flatpak does not. 

At some point I had changed the qt theme to kvantum in an effort to make it use the same theme at gtk.

I just tried changing the qt theme to anything else, and the freezing stops in the newest build while using any qt theme beside kvantum and kvantum-dark. I now have it set to "gtk2", which also works to make themes uniform. I don't remember why I had opted to use kvantum instead. 

Still not sure why the freezing didn't occur (at least as reliably) in older builds.
